superset-frontend/src/SqlLab/components/ResultSet/index.tsx

```diff
@@ -70,14 +70,6 @@ interface ResultSetProps {
   defaultQueryLimit: number;
 }
 
-interface ResultSetState {
-  searchText: string;
-  showExploreResultsButton: boolean;
-  data: Record<string, any>[];
-  showSaveDatasetModal: boolean;
-  alertIsOpen: boolean;
-}
-
 const ResultlessStyles = styled.div`
   position: relative;
   min-height: 100px;
```

Comment:
```suggestion
  min-height: ${({ theme }) => theme.gridUnit * 25}px;
```
Unit values should stay within the Superset theme

Reply:
I can't target this spot for another "suggested change" comment, but the values for `font-size` and `line-height` in `ReturnedRows` on [these lines](https://github.com/apache/superset/blob/7d21ad4a1124c81d076906fbc76538995372e008/superset-frontend/src/SqlLab/components/ResultSet/index.tsx#L95-L96) also need to be converted to use the Superset theme